iothub/device/src/IotHubBaseClient.cs

```diff
@@ -328,7 +342,12 @@ public async Task<long> UpdateReportedPropertiesAsync(ReportedProperties reporte
         /// This has the side-effect of subscribing to the PATCH topic on the service.
         ///  </para>
         /// </remarks>
-        /// <param name="callback">The callback to be invoked when a desired property update is received from the service.</param>
+        /// <param name="callback">
+        /// The callback to be invoked when a desired property update is received from the service.
+        /// This user-supplied callback is "fire-and-forget" and the SDK doesn't wait on it. All of requests will be processed as they arrive.
+        /// Exceptions will be thrown to the users and the SDK won't log them internally, so the user is responsible
```

Comment:
Same question here about the meaning of "Exceptions will be thrown to the users".

Reply:
we don't catch or handle any exceptions in the method `OnDesiredStatePatchReceived()`